pinot-controller/src/main/java/org/apache/pinot/controller/api/resources/Constants.java

```diff
@@ -30,6 +30,7 @@
 
   public static final String CLUSTER_TAG = "Cluster";
   public static final String TABLE_TAG = "Table";
+  public static final String CONFIG_TAG = "Config";
```

Comment:
`Config` seems too general. I would suggest putting these new endpoints under `Table` and use path a different path, e.g. `/tableconfigs`

Reply:
Done.